src/main/scala/com/codacy/docker/api/utils/FileHelper.scala

```diff
@@ -33,7 +33,7 @@ object FileHelper {
 
   def findConfigurationFile(candidates: Set[String], path: Path): Option[Path] = {
     candidates.flatMap { nativeConfigFileName =>
-      better.files.File(path).listRecursively.filter(f => f.name == nativeConfigFileName).map(_.path)
+      listAllFiles(path).filter(_.getName == nativeConfigFileName).map(_.toPath)
```

Comment:
I would prefer to use better files to do this instead of using our implementation.
Less code, less tests needed, less maintenance.